src/plot_api/to_image.js

```diff
@@ -128,7 +140,6 @@ function toImage(gd, opts) {
     // extend config for static plot
     var configImage = Lib.extendFlat({}, config, {
         staticPlot: true,
-        plotGlPixelRatio: config.plotGlPixelRatio || 2,
```

Comment:
Ooops. This should have been in a separate commit. In brief, this line was useless, so :hocho: